dask_jobqueue/core.py

```diff
@@ -351,6 +352,8 @@ def __init__(
         if worker_extra_args is not None:
             command_args += worker_extra_args
 
+        # Save command args to be extended, etc.
+        self._command_args = command_args
```

Comment:
I'm not sure why you add this line, do you use it anywhere else?

Reply:
I had an initial design that did (and removed it) - will remove this line too.